### DISCUSSION TITLE
Documentation: Consider referencing pthom's imgui_manual

### DISCUSSION THREAD

**Discussion by jhgorse** [ocornut/imgui#4608] [New Users Build/Link/Run issues ONLY!]
Greetings,

I love the library and the simple joy it brings to my software. It reminds me of when I started and things Just Worked (tm). =)

I have a recommendation. Consider linking to @pthom's imgui_manual, front and center, in the Demo section: https://github.com/ocornut/imgui#demo

Here is a link to his Emscripten build which demonstrates Dear ImGui live and with references to the demo C++ code it is running and you are inspecting in real time:
https://pthom.github.io/imgui_manual_online/manual/imgui_manual.html

Cheers,
Joe Gorse

**Comment by ocornut** [comment]
There's a link from https://github.com/ocornut/imgui/wiki#demo-examples
But you are right, will add more links in the meanwhile.
Note that we are expecting to merge this (or part of) in #3689
